### ISSUE TITLE
leave a message with a profile when not logged in

### ISSUE THREAD

**Issue by scd** [theCrag/website#1842]
The leave a message button should still be available on a profile when not logged in. When clicked you should be prompted to log in.

**Comment by brendanheywood** [comment]
yup a general thing across the board, actions which only rely on being logged in (as opposed to say coordinator role) should always be visible and click-able and prompt for login on the way.

**Comment by brendanheywood** [comment]
Ahh we can't do this because the message compose page contains the current user:

https://brendan.thecrag.com/climber/brendanheywood/messages/compose?toid=123873942&to=benyv

Which we don't know until after you've logged in. This is a much deeper bug than I thought, ultimately I really want to re-route that url so it is simply:

https://brendan.thecrag.com/messages/compose?toid=123873942&to=benyv

I've removed the broken button for now
